pkg/network/setup/netconf_test.go

```diff
@@ -76,61 +69,6 @@ var _ = Describe("netconf", func() {
 		netConf := netsetup.NewNetConfWithCustomFactory(nil, failingCacheCreator{})
 		Expect(netConf.Teardown(vmi)).NotTo(Succeed())
 	})
-
-	Context("with completion cache", func() {
```

Comment:
Now that I see these tests removed at the netconf level, how do we check that the same cache is re-used on setup? And that teardown does not leak it?